node/external/interface.go

```diff
@@ -1,8 +1,16 @@
 package external
 
+import (
+	"github.com/ElrondNetwork/elrond-go/process/smartContract"
+	vmcommon "github.com/ElrondNetwork/elrond-vm-common"
+)
+
+// TODO: Move interface to scDataGetter.go
+// TODO: Rename ScDataGetter to "SmartContractRunSimulator"
```

Comment:
it is not necessary a simulator. in real life scenario it is used to get the values from a certain smart contract state. like getting my erc20 balance.